spec/game_spec.rb

```diff
@@ -0,0 +1,26 @@
+
+require './lib/game'
+RSpec.describe Game  do
+	it "the game must be initialize with default value" do
+		g = Game.new
+		expect(g.b).to eql(nil)
+	    expect(g.turn).to eql("x")
+	    expect(g.score).to eql([0,0,0])
+	end	
+	describe "switch_turn" do
```

Comment:
Are you using the turn variable for something? If not, what's the point of testing it?